cli/popper/commands/cmd_dot.py

```diff
@@ -90,6 +90,11 @@ def add_to_graph(dot_str, wf, parent, children, node_attrs, stage_edges):
         wfile_list.append(pu.find_default_wfile(wfile))
 
     for wfile in wfile_list:
+        if wfile.endswith('.workflow'):
```

Comment:
we can create a static factory method of `Workflow` that has this logic so that we don't need to repeat it every time we want to instantiate a `Workflow` object? Something like `Workflow.new_workflow(filename)`?

Reply:
Ah. Yes. This looks much cleaner.